### DISCUSSION TITLE
any way to add looping audio to web wallpaper?

### DISCUSSION THREAD

**Discussion by Hinge45** [rocksdanister/lively#1169] [General]
Can't find a way to add a looping rain sound to the rain wallpaper.

**Comment by dabjulmaros** [comment]
Both video and audio tags have loop attributes 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-loop
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio#attr-loop

**Reply by Hinge45** [reply]
it doesn't work. (autoplay won't work for some reason)

**Reply by dabjulmaros** [reply]
Ha you right autoplay doesn't work with videos that are not muted.
There is a way to change that policy in chrome, https://stackoverflow.com/questions/49921453/how-to-allow-video-autoplay-in-a-google-chrome-kiosk-app-in-version-66-or-later#:~:text=opening%20a%20Chrome%20browser%20and,rebooting%20the%20device
I'm not sure if lively currently supports changing the settings or flags required to autoplay videos with sound.

**Reply by rocksdanister** [reply]
That's weird.. Its already set:
https://github.com/rocksdanister/lively-cef/blob/55390e45738ffe10384b9d83f2aa20ef7b49ed89/src/Lively.PlayerCefSharp/Form1.cs#L376
